src/CaBundle.php

```diff
@@ -82,15 +82,13 @@ public static function getSystemCaRootBundlePath(?LoggerInterface $logger = null
 
         $otherLocations = array(
             '/etc/pki/tls/certs/ca-bundle.crt', // Fedora, RHEL, CentOS (ca-certificates package)
-            '/etc/ssl/certs/ca-certificates.crt', // Debian, Ubuntu, Gentoo, Arch Linux (ca-certificates package)
+            '/etc/ssl/certs/ca-certificates.crt', // Debian, Ubuntu, Gentoo, Arch Linux (ca-certificates package), FreeBSD (only the /etc/ssl/certs dir which gets added below)
```

Comment:
This really read awkward. It looks rather like a hack than a fix. Maybe a test in the look shall determine whether `otherLocations` entry is a dir or file?

Reply:
Ok I tweaked it but this will remove a few lookups, which I think is probably fine.. Hopefully it doesn't cause regressions.

Reply:
Thank you, merged appreciated. As soon as this lands in composer I will create a downstream PR to remove the depencency to ca_root_nss in FreeBSD.

Reply:
> Composer 2.8.3 ships with this PR

Magic, thank you! I will create the downstream request with the FreeBSD port to remove the dependency on ca_root_nss!

Reply:
Downstream request: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=282816